gradle-baseline-java/src/test/groovy/com/palantir/baseline/BaselineFormatIntegrationTest.groovy

```diff
@@ -154,23 +192,4 @@ class BaselineFormatIntegrationTest extends AbstractPluginTest {
         BuildResult result = with('spotlessJavaCheck').build()
         result.task(":spotlessJava").outcome == TaskOutcome.SUCCESS
     }
-
-    def 'eclipse format trims blank lines in block or javadoc comment'() {
-        when:
-        buildFile << standardBuildFile
-        file('gradle.properties') << """
-            com.palantir.baseline-format.eclipse=true
-        """.stripIndent()
-        file('src/main/java/test/Test.java').text = resourceAsString("blank-lines-in-comments.java")
-
-        then:
-        BuildResult result = with('format').build()
-        result.task(":spotlessJavaApply").outcome == TaskOutcome.SUCCESS
-        file('src/main/java/test/Test.java').text == resourceAsString("blank-lines-in-comments-fixed.java")
-    }
```

Comment:
this test and all others that don't test unusual conditions (e.g. different source set, different langauge) has now been moved to the "eclipse formatter integration test"